spec/classes/init_spec.rb

```diff
@@ -34,15 +43,24 @@
       host_factory_token: sensitive('the host factory token'),
     } end
 
-    it "creates the host using the host factory" do
+    before do
       allow_calling_puppet_function(:'conjur::manufacture_host', :create) \
-          .with(include('uri' => 'https://conjur.test/api'), 'test', sensitive('the host factory token'))\
+          .with(include('uri' => 'https://conjur.test/api/'), 'test', sensitive('the host factory token'))\
           .and_return 'api_key' => sensitive('the api key')
       allow_calling_puppet_function(:'conjur::token', :from_key) \
-          .with(include('uri' => 'https://conjur.test/api'), 'host/test', sensitive('the api key'))\
+          .with(include('uri' => 'https://conjur.test/api/'), 'host/test', sensitive('the api key'))\
           .and_return sensitive('the token')
+    end
+
+    it "creates the host using the host factory" do
       expect(lookupvar('conjur::token')).to eq sensitive('the token')
     end
+
+    it "stores the configuration and identity on the node" do
+      expect(subject).to contain_file('/etc/conjur.conf')
+      expect(subject).to contain_file('/etc/conjur.identity').with_content \
+          %r(machine https://conjur.test/api/authn\s+login host/test\s+password the api key)
```

Comment:
I thought about that, but I figured it'd be verifying what's just declared. Maybe you're right, though: better safe than sorry in this case. I'll add it.